util/log/file.go

```diff
@@ -38,9 +39,41 @@ import (
 // MaxSize is the maximum size of a log file in bytes.
 var MaxSize uint64 = 1024 * 1024 * 10
 
-// If non-empty, overrides the choice of directory in which to write logs.
-// See createLogDirs for the full list of possible destinations.
-var logDir *string
+// If non-empty, overrides the choice of directory in which to write logs. See
+// createLogDirs for the full list of possible destinations. Note that the
+// default is to log to stderr independent of this setting. See --logtostderr.
+var logDir = os.TempDir()
+var logDirSet bool
+
+// DirSet returns true of the log directory has been changed from its default.
+func DirSet() bool {
+	return logDirSet
+}
+
+type stringValue struct {
```

Comment:
Perhaps we should start moving these custom flag values to their own package, cli/flags or something like that.  StoreSpec is in server, this one is here.  Might be better to keep them together.
(not for this PR, but a subsequent one)